prompts/template_xml/corpus_generation_via_python_script.txt

```diff
@@ -0,0 +1,23 @@
+Hi! I'm a security engineer that is looking to improve the quality of my fuzzing harness.
+
+The goal is to look at how the corpus of the fuzzing harness can be improved.
+
+The project is {PROJECT_NAME} and the repository of the project is {PROJECT_REPOSITORY}.
+
+The fuzzer I'm working on is:
+<code>
+{HARNESS_SOURCE_CODE}
+</code>
+
+The target code of the harness is:
+<code>
+{TARGET_FUNCTION_SOURCE}
+</code>
+
+Could you please construct a small Python program that generate 10 initial seeds for my fuzz harness?
```

Comment:
How about also generating a dictionary? It can improve the efficiency of mutation. For example, if fuzzing cannot tell which of two paths will have higher coverage, a dictionary will increase the likelihood of generating inputs for the desired one.
Similarly, suppose LLM needs to encode some path constraints into the seed (e.g., `if (x=="ABC")`, but `x` is dynamically extracted from a non-fixed part of the input), a dictionary provides a more flexible way to increase the chance of generating that value.